ingestion_tools/scripts/data_validation/fixtures/path.py

```diff
@@ -305,12 +305,28 @@ def tomo_zarr_file(
     return file
 
 
+# =============================================================================
+# Run and voxel-specific fixtures, Neuroglancer
+# =============================================================================
+
+
 @pytest.fixture(scope="session")
-def tomo_basename(
-    tomo_zarr_file: str,
+def neuroglancer_dir(tomo_dir: str, filesystem: FileSystemApi) -> str:
+    """[Dataset]/[ExperimentRun]/Tomograms/VoxelSpacing[voxel_spacing]/CanonicalTomogram/"""
+    return tomo_dir
+
```

Comment:
This is very nice, as it gives us opportunity to easily make any updates in the future.